examples/3.0.0/channel.json

```diff
@@ -0,0 +1,40 @@
+[{
```

Comment:
I wonder why this is an array

Reply:
This is an array because we can pass multiple channel examples. Also the example properties can only be an array